pkg/virt-handler/heartbeat/heartbeat.go

```diff
@@ -93,7 +101,13 @@ func (h *HeartBeat) labelNodeUnschedulable() (done chan struct{}) {
 			v1.CPUManager, cpuManagerEnabled,
 			v1.VirtHandlerHeartbeat, string(now),
 		))
-		_, err = h.clientset.Nodes().Patch(context.Background(), h.host, types.StrategicMergePatchType, data, metav1.PatchOptions{})
+
+		// TODO: This can be safely removed once we do not support upgrade from a version that does not have Shadow Node
+		_, err = h.clientSet.Patch(context.Background(), h.host, types.StrategicMergePatchType, data, metav1.PatchOptions{})
```

Comment:
Should this be controlled by nodeBackoff flag as well?

Reply:
`nodeBackoff` is meant only to decrease #writes. This function is called rarely and therefore I don't think we need to include it.